apps/explorer/src/lib/utils/sentry.ts

```diff
@@ -33,25 +33,12 @@ export function initSentry() {
             }),
         ],
         tracesSampleRate: SENTRY_SAMPLE_RATE,
-        beforeSend(event) {
-            try {
-                // Filter out any code from unknown sources:
-                if (
-                    !event.exception?.values?.[0].stacktrace ||
-                    event.exception?.values?.[0].stacktrace?.frames?.[0].filename === '<anonymous>'
-                ) {
-                    return null;
-                }
-                // eslint-disable-next-line no-empty
-            } catch (e) {}
-
-            return event;
-        },
 
         denyUrls: [
-            // Chrome extensions
+            // Browser extensions
```

Comment:
```suggestion
            denyUrls: [
              // Browser extensions
              /extensions\//i,
              /^chrome(?:-extension)?:\/\//i,
              /^moz-extension:\/\//i,
              /^safari-(?:web-)?extension:\/\//i,
              /^edge:\/\//i,
              /<anonymous>/,
          ],
```

Our users using safari & edge, it might be better to filter their extensions too